index.html

```diff
@@ -1,3 +1,29 @@
-<html>
-  
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <ul class="nav-bar">
+            <li><a href="#">Porfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">info</a></li>
+        </ul>
+    </header>
+    <main class="main-content">
+        <h1 class="heading">Hey there I,m Bunbee.<br>I'm a software developer</h1>
+        <p>I can help you build a product, feature or website. Look thrrough some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+        <ul class="footer">
+            <li><a href="https://github.com/mutinhiri">github</a></li>
+            <li><a href="www.linkedin/mutinhiri">Linkedin</a></li>
+            <li><a href="twitter.com/bunbee">Twitter</a></li>
+        </ul>
```

Comment:
- [ ] This is a good start but instead of names they should be icons. [see original links](https://user-images.githubusercontent.com/61562626/129913663-13a20daa-8aa9-4e70-b32d-1c530bf2ca49.jpeg) and your
![bvbvvbvb](https://user-images.githubusercontent.com/61562626/129913814-6a706678-143c-4ba0-9bb4-4bf49b26ace2.jpeg)